contract_manager/scripts/list_wormhole_contracts.ts

```diff
@@ -0,0 +1,40 @@
+import yargs from "yargs";
+import { hideBin } from "yargs/helpers";
+import { DefaultStore, EvmWormholeContract } from "../src";
+
+const parser = yargs(hideBin(process.argv))
+  .usage("Usage: $0")
+  .options({
+    testnet: {
+      type: "boolean",
+      default: false,
+      desc: "Fetch testnet contracts instead of mainnet",
+    },
+  });
+
+async function main() {
+  const argv = await parser.argv;
+  const entries = [];
+  for (const contract of Object.values(DefaultStore.wormhole_contracts)) {
+    if (contract instanceof EvmWormholeContract) {
+      if (contract.getChain().isMainnet() === argv.testnet) continue;
+
+      try {
+        const index = await contract.getCurrentGuardianSetIndex();
+        const chainId = await contract.getChainId();
```

Comment:
In general it's preferable to do `const [index, chainId] = await Promise.all([ contract.getCurrentGuardianSetIndex(), contract.getChainId() ]);` so that the async calls can be parallelized